python/graphscope/framework/graph.py

```diff
@@ -154,7 +183,30 @@ def _construct_op_of_empty_graph(self):
 
 
 class GraphDAGNode(DAGNode, GraphInterface):
-    """Graph node in a DAG."""
+    """A class represents a graph node in a DAG.
+
+    In GraphScope, all operations that generate a new graph will return
+    a instance of :class:`GraphDAGNode`, which will be automatically
+    executed by :method:`sess.run` in `eager` mode.
+
+    The following example demonstrates its usage:
+
+    .. code:: python
+
+        >>> # lazy mode
+        >>> import graphscope as gs
+        >>> sess = gs.session(mode="lazy")
+        >>> g = sess.g()
+        >>> g1 = g.add_vertices("person.csv","person")
+        >>> sess.run(g1)
+
+        >>> # eager mode
+        >>> import graphscope as gs
+        >>> sess = gs.session(mode="eager")
+        >>> g = sess.g()
+        >>> g1 = g.add_vertices("person.csv","person")
```

Comment:
Better tell user the difference between the two `add_vertices`, i.e. you can print then out.